build.sh

```diff
@@ -16,7 +16,7 @@
 set -exv
 
 # retrieve the latest tag set in repository
-version=$(git describe --tags --abbrev=0)
+version=$(git describe --always --tags --abbrev=0)
```

Comment:
what it does pls?

Reply:
resolved in slack

Reply:
can you point me to the discussion please?